### PR TITLE
added more effects

### PR DESCRIPTION
**Pull request by MaxVoloskiy**
Added all effects to our app and created functionality for choosing visibility for each effect.

### CODE REVIEW

File: src/client/assets/icons/icons.js [hunk 1]
```diff
@@ -28,3 +29,4 @@ export const stopIcon = <FontAwesomeIcon icon={faStop} className="icon" />;
 export const playIcon = <FontAwesomeIcon icon={faPlay} className="icon" />;
 export const pauseIcon = <FontAwesomeIcon icon={faPause} className="icon" />;
 export const cancelWindowIcon = <FontAwesomeIcon icon={faTimes} />;
+export const checkTick = <FontAwesomeIcon icon={faCheck} />;
```
Comment (StanislavFedyna) [inline src/client/assets/icons/icons.js]: use a classic type of naming icons please, like checkTickIcon)

File: src/client/components/allBlocks/allBlocks.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import connect from 'react-redux/es/connect/connect';
+import {
+  setVisibility,
+} from '../../actions/blocksActions';
+import BlockOfSliders from '../blockOfSliders/index';
+import Button from '../button/index';
```
Comment (StanislavFedyna) [inline src/client/components/allBlocks/allBlocks.js]: Why you call index file, if webpack get this file by default)

File: src/client/components/allBlocks/allBlocks.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+            }, i) => {
+              if (isVisible) {
+                return (
+                  <BlockOfSliders
+                    name={name}
+                    effects={effects}
+                    createEffect={createEffect}
+                    key={i}
+                    sound={sound}
+                    setEffectsValue={this.setEffectsValue}
```
Comment (StanislavFedyna) [inline src/client/components/allBlocks/allBlocks.js]: I think you can use destructuring in this case (for setEffectsValue)

File: src/client/assets/icons/icons.js (hunk 1 above)
Comment (MaxVoloskiy) [reply src/client/assets/icons/icons.js]: fixed

File: src/client/components/allBlocks/allBlocks.js (hunk 2 above)
Comment (MaxVoloskiy) [reply src/client/components/allBlocks/allBlocks.js]: fixed

File: src/client/components/allBlocks/allBlocks.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+                <button
+                  className="Effect"
+                  key={i}
+                  type="button"
+                  id={name}
+                  name={name}
+                  onClick={() => setVisibility(name)}
+                >
+                  {name}
+                  {' '}
```
Comment (Kindest13) [inline src/client/components/allBlocks/allBlocks.js:108]: what is that?

**Review by StanislavFedyna — DISMISSED** [review]

File: src/client/components/allBlocks/allBlocks.js (hunk 4 above)
Comment (MaxVoloskiy) [reply src/client/components/allBlocks/allBlocks.js:108]: Space

**Comment by tonypntsrn** [comment]
Please, give a brief description what have been done in scope of this PR

File: src/client/components/allBlocks/allBlocks.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+                  {isVisible ? checkTickIcon : ''}
+                </button>
+              ))
+            }
+          </div>
+          <div className="Sliders">
+            {blocksData.map(({
+              name, effects, createEffect, isVisible,
+            }, i) => {
+              if (isVisible) {
```
Comment (tonypntsrn) [inline src/client/components/allBlocks/allBlocks.js]: You could use ternary operator here

**Comment by MaxVoloskiy** [comment]
> Please, give a brief description what have been done in scope of this PR

added

File: src/client/components/allBlocks/allBlocks.js (hunk 5 above)
Comment (MaxVoloskiy) [reply src/client/components/allBlocks/allBlocks.js]: done

File: src/client/components/allBlocks/allBlocks.js [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+    return (
+      <div className="SlidersComponent__main--container">
+        <div className="AllSliders">
+          <div className="ListOfEffects">
+            {
+              blocksData.map(({ name, isVisible }, i) => (
+                <button
+                  className="Effect"
+                  key={i}
```
Comment (NadiiaChorna) [inline src/client/components/allBlocks/allBlocks.js]: It's not a good practise to set index as the keys in "map" expression. I would suggest to use "name" property instead.

Comment (MaxVoloskiy) [reply src/client/components/allBlocks/allBlocks.js]: Got it. Fixed

**Review by skiffua — DISMISSED** [review]

File: src/client/components/allBlocks/allBlocks.js [hunk 7]
```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import connect from 'react-redux/es/connect/connect';
```
Comment (StanislavFedyna) [inline src/client/components/allBlocks/allBlocks.js]: bad import with connect
You know what I want )

**Review by StanislavFedyna — APPROVED** [review]

**Review by tonypntsrn — APPROVED** [review]

**Pull request merged**